corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -931,26 +931,35 @@ hqDefine("cloudcare/js/form_entry/entries", [
         // Tracks whether file entry has already been cleared, preventing an additional failing request to Formplayer
         self.cleared = false;
         self.buildBroadcastTopics(options);
+        self.uploadFile = function() {
+            document.getElementById(self.entryId).click();
+        }
+        self.fileNameDisplay = ko.observable(constants.NO_FILE_SELECTED)
```

Comment:
translation here?

Reply:
I'm not super familiar with how to enable strings in JS to be translatable - is there a JS equivalent to Python's `gettext` that we frequently use?

Reply:
You can use `gettext` https://commcare-hq.readthedocs.io/translations.html#tagging-strings-in-javascript